packages/@aws-cdk/aws-servicecatalog/rosetta/default.ts-fixture

```diff
@@ -0,0 +1,12 @@
+// Fixture with packages imported, but nothing else
+import { Construct, Stack } from '@aws-cdk/core';
+import servicecatalog = require('@aws-cdk/aws-servicecatalog');
+import iam = require('@aws-cdk/aws-iam');
```

Comment:
Use `import * as xyz from 'xyz'` syntax instead here:

```
import * as servicecatalog from '@aws-cdk/aws-servicecatalog';
import * as iam from '@aws-cdk/aws-iam';
```

Reply:
Is there better guidance docs here?  I copied this formatting from the `aws-ec2`  cdk README.

Reply:
Unfortunately, not really - it's more of a convention we use. I have no idea why does the EC2 module stray away from it, my best guess would be it's just a mistake.